<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>云备忘</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/YFReminder2.svg</normaloff>:/icon/YFReminder2.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblTagToday">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSingleWeek">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledReminder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoteSync">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>云上传</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoteDownload">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>云下载</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
